src/sentry/static/sentry/less/stream.less

```diff
@@ -541,10 +541,18 @@
     line-height: 24px;
   }
 
+
   .event-graph {
-    .sparkline a {
+    .sparkline, .barchart, a {
       height: 24px;
     }
+    .zero-bar {
+      background-color: transparent;
+    }
+    .barchart {
```

Comment:
is this to draw the line underneath it? why this and not `border-bottom: 1px solid @gray-lightest`?